exporter/redis.go

```diff
@@ -139,6 +141,26 @@ var (
 		// # Cluster
 		"cluster_stats_messages_sent":     "cluster_messages_sent_total",
 		"cluster_stats_messages_received": "cluster_messages_received_total",
+
+		// # Tile38
+		// based on https://tile38.com/commands/server/
+		"aof_size":        "aof_size_bytes",
+		"avg_item_size":   "avg_item_size_bytes",
+		"cpus":            "cpus_total",
+		"heap_released":   "heap_released_byte",
+		"heap_size":       "heap_size_bytes",
+		"http_transport":  "http_transport",
```

Comment:
Not familiar with Tile38 - what does this metric store/expose?

Reply:
You can see [here](https://tile38.com/commands/server/) to find out more about tile38 metrics.

Reply:
I don't see `http_transport` on that page - what is that one actually exporting?